hack/functests.sh

```diff
@@ -68,9 +68,17 @@ if [ "$KUBEVIRT_E2E_PARALLEL" == "true" ]; then
         serial_test_args="${serial_test_args} --focus=\\[Serial\\]"
     fi
 
+    return_value=0
+    set +e
     functest --nodes=${KUBEVIRT_E2E_PARALLEL_NODES} ${parallel_test_args} ${FUNC_TEST_ARGS}
+    return_value="$?"
+    set -e
+    if [ "$return_value" -ne 0 ] && ! [ "$KUBEVIRT_E2E_RUN_ALL_SUITES" == "true" ]; then
+        exit "$return_value"
+    fi
     extra_args="-junit-output ${ARTIFACTS}/partial.junit.functest.xml"
     functest ${serial_test_args} ${FUNC_TEST_ARGS}
+    exit "$return_value"
```

Comment:
Can it be that we would now exit wit a success here if the parallel tests succeed, but the serial ones do not?

Reply:
No, because the serial tests would exit with a non zero code if there were test failures. That's also the reason why the serial tests were skipped when there were test failures in the parallel test suite.

Reply:
:+1: